README.adoc

```diff
@@ -51,23 +51,27 @@ The plugin provides bootstrap script that creates sample project with:
 * proper source file layout
 * sample https://gatling.io/docs/current/general/simulation_structure/[Simulation class], demonstrating sufficient `Gatling` functionality
 
-For this quickstart - `git` and `curl` must be instlled and available in `$PATH`.
+For this quickstart - `git` and `curl` must be installed and available in `$PATH`.
 
-.Generate sample Gradle project and run Gatling simulation
-[source, bash]
-----
-curl -sL https://git.io/Jf2Uk | bash -s ~/sample-gradle-gatling
-----
 
-.Navigate to new project folder and run all simulations
+[TIP]
+====
+Sample project skeleton should be adjusted after a new plugin will be released.
+====
+
+.Generate sample Gradle project and run Gatling simulation, follow on-screen instructions.
 [source, bash]
 ----
-cd ~/sample-gradle-gatling
-./gradlew gatlingRun
+curl -sL https://git.io/JJj3A | bash -s ~/gradle-gatling-sample
```

Comment:
Matter of taste (and security), but I personally really dislike the `curl | bash` template.

Too bad that gradle does not provide a template/g8/archetype system.

Perhaps should we provide an archive (`zip` and `tgz`) containing the sample project?

At least, we won't use an URL shortener.

Reply:
Don't see any good reasons to edit \ change this PR if the entire idea of shell script is going be be rejected. It's better to decline PR instead.

Reply:
As I said, it is a matter of (my) personal taste. the shell script won't be rejected.

But I think we need to point for the template directory (by cloning this repo?) as well to let the user choose.
If we can provide an archive, it can be another alternative.

Reply:
> But I think we need to point for the template directory (by cloning this repo?)

The script does clone this repo already. Partially, only template directory `src/main/bootstrap`

> as well to let the user choose.

let choose _what_ ?

Reply:
This thread is about the `curl | bash` pattern.
So even if the script is actually clone (partially) the repo, I suggest documenting to let the user choose between executing the dark but convenient `curl | bash` pattern or manual clone the repo.

Reply:
Seems you' misunderstood a little bit
There's no option in this PR to 

> _manually cloning the repo_  

The only way to generate sample - is to run some script. This is a core of PR.

Reply:
Seems we don't succeed to communicate in either way ^^

I suggested to add documentation to have another choice instead of only the `curl | bash` pattern.

I agree that it is a convenient pattern, often used in the wild internet. But I would like to see another way documented as well: either the manual cloning or the archive (include the "why not both?" meme here).

If I try to reformulate:
Can you add the documentation to bypass the `curl | bash` pattern, please?

Reply:
One more time.

The only way to create a sample repository, provided by this PR is - to use script described in README. There's no other options, provided by this PR. So, I am unable to add documentation you had requested.

Reply:
To achieve what you want, it is required to add a publish step to pipeline, that generates sample project TGZ and publishes it together with GitHub release.

But this obviously is not part of this PR.

Reply:
I thought that the aim of this PR was to "provide a sample project" and the script is only a convenient way to get it.

It was not clear to me that it was in fact: "provide a script to generate a sample project"

I agree that the archive generation is not mean to be part of this PR.

I thought, however, that documenting the full manual steps could have been part.

Ok. I let that part aside from now.